Lib/fontTools/misc/plistlib.py

```diff
@@ -410,6 +423,16 @@ def totree(
 
 
 def fromtree(tree, use_builtin_types=None, dict_type=dict):
+    """Converts an XML tree to a plist structure.
+
+    Args:
+        tree: An ``etree`` ``Element``.
+        use_builtin_types: Whether to use standard plist types or
+            to wrap values in a Data block. Deprecated.
+        dict_type: What type to use for dictionaries.
+
+    Returns: A dictionary which can be serialized as a plist.
```

Comment:
This doesn’t necessarily return a dictionary. The return type is whatever is appropriate for the plist.

Reply:
> The return type is whatever is appropriate for the plist.

correct. Maybe just say "a plist object"

Reply:
Other docstrings say “An object (usually a dictionary)”.

Reply:
sounds good